cygnus-twitter/README.md

```diff
@@ -1 +1,296 @@
-TBD
+#<a name="top"></a>Cygnus-twitter
+[![License badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
+[![Documentation badge](https://readthedocs.org/projects/fiware-cygnus/badge/?version=latest)](http://fiware-cygnus.readthedocs.org/en/latest/?badge=latest)
+[![Docker badge](https://img.shields.io/docker/pulls/fiware/cygnus.svg)](https://hub.docker.com/r/fiware/cygnus/)
+[![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware-cygnus)
+[![Support badge]( https://img.shields.io/badge/support-askbot-yellowgreen.svg)](https://ask.fiware.org/questions/scope%3Aall/tags%3Acygnus/)
+
+* [Welcome to Cygnus-twitter](#section1)
+* [Basic operation](#section2)
+    * [Hardware requirements](#section2.1)
+    * [Configuration](#section2.3)
+    * [Unit testing](#section2.5)
+    * [Management API overview](#section2.7)
+* [Features summary](#section5)
+* [Licensing](#section6)
+* [Reporting issues and contact information](#section7)
+
+##<a name="section1"></a>Welcome to Cygnus-twitter
+This project is part of [FIWARE](http://fiware.org), being part of the [Cosmos](http://catalogue.fiware.org/enablers/bigdata-analysis-cosmos) Ecosystem.
+
+Cygnus-twitter is a connector in charge of persisting tweets (https://dev.twitter.com/overview/api/tweets) in certain configured third-party storages, creating a historical view of such data.
+
+Internally, Cygnus-twitter is based on [Apache Flume](http://flume.apache.org/) and [Twitter4j](http://twitter4j.org/en/index.html). In fact, Cygnus-twitter is a Flume agent, which is basically composed of a source in charge of receiving the data from Twitter (https://twitter.com/), a channel where the source puts the data once it has been transformed into a Flume event, and a sink, which takes Flume events from the channel in order to persist the data within its body into a third-party storage.
+
+Current stable release is able to persist Twitter data in:
+
+* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
+
+You may consider to visit [Cygnus-twitter User and Programmer Guide](doc/cygnus-twitter/user_and_programmer_guide/README.md) before going deep into the details.
```

Comment:
I think this paragraph was originally intenteded for linking to a quick start guide. If such a quick start guide does not exist, simply remove this line ;)